### DISCUSSION TITLE
Allow for download spec to be JSON or YAML based on config

### DISCUSSION THREAD

**Discussion by marclave** [scalar/scalar#733] [Ideas]
Scalar users should be able to decide on if the download spec format is JSON or YAML ✨  through a config parameter

**Comment by hanspagel** [comment]
I think it’s the same as the input, isn’t it?

Ah, maybe that was changed in one of the recent refactors.

**Reply by marclave** [reply]
it is, however with our integrations in frameworks our users have less control on the format of spec that gets fed in (dependent on the framework). so if we have an override somwhere it let's our users have more control :)
